File: IOT/Anshul-13J_AnshulNigam_2024cs1069_2/FunctionAndArrays/SpanOfArray.cpp
#include<iostream>
using namespace std;

int main(){
    int n,max,min;
    cin>>n;
    int a[n],i;
    for(i=0;i<n;i++)
    {
        cin>>a[i];
    }
    max=a[0];   min=a[0];
    for(i=0;i<n;i++)
    {
        if(a[i]>max)
            max=a[i];
        if(a[i]<min)
            min=a[i];
    }
    cout<<max-min;
}

File: Android/Tejash11_TejashSeth_2024csit1065_2/FunctionAndArrays/SpanOfArray.java
import java.io.*;
import java.util.*;

public class Main{

public static void main(String[] args) throws Exception {
   Scanner sc = new Scanner(System.in);
   int n = sc.nextInt();
   int a[]= new int[n];
   for(int i=0;i<n;i++)
   {
       a[i]=sc.nextInt();
   }
   int min=a[0],max=a[0];
   for(int i=0;i<n;i++)
   {
       if(a[i]<min)
       min=a[i];
       if(a[i]>max)
       max=a[i];
   } 
   System.out.println(max-min);
}
}